Packs/CommonScripts/Scripts/ExtractAttackPattern/ExtractAttackPattern_test.py

```diff
@@ -1,5 +1,7 @@
 from ExtractAttackPattern import is_valid_attack_pattern
+
 from CommonServerPython import *
+import ExtractAttackPattern as eap
 
 
 def test_extract_existing_mitre_ids(mocker):
```

Comment:
how this test did not catch the bug?

Reply:
Because I mock the executeCommand with incorrect return value:
![image](https://user-images.githubusercontent.com/57212002/141938032-c797a753-c1c8-4ede-8c14-533d1827975d.png)